src/ReasonReactRouter.re

```diff
@@ -145,13 +163,16 @@ let urlNotEqual = (a, b) => {
         listNotEqual(aRest, bRest);
       }
     };
-  };
   a.hash !== b.hash || a.search !== b.search || listNotEqual(a.path, b.path);
 };
 type watcherID = unit => unit;
-let url = () => {path: path(), hash: hash(), search: search()};
+let url = (~serverUrlString=?, ()) => {
+  path: path(~serverUrlString, ()),
+  hash: hash(~serverUrlString, ()),
+  search: search(~serverUrlString, ()),
+};
 /* alias exposed publicly */
-let dangerouslyGetInitialUrl = url;
+let dangerouslyGetInitialUrl = () => url();
```

Comment:
Pass `serverUrlString` here.

Reply:
ahhh, okay I'll use this instead of `fromServer` also, will update tests to call this instead.